packages/autorest.gotest/test/integrationtest/output/appplatform/armappplatform/configservers_client_example_test.go

```diff
@@ -66,11 +83,27 @@ func ExampleConfigServersClient_BeginUpdatePut() {
 	if err != nil {
 		log.Fatalf("failed to pull the result: %v", err)
 	}
-	// TODO: use response item
 	_ = res
+	// If the HTTP response code is 200 as defined in example definition, your response structure would look like this. Please pay attention that all the values in the output are fake values for just demo purposes.
+	// res.ConfigServerResource = armappplatform.ConfigServerResource{
+	// 	Name: to.Ptr("default"),
+	// 	Type: to.Ptr("Microsoft.AppPlatform/Spring/configServers"),
+	// 	ID: to.Ptr("/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myResourceGroup/providers/Microsoft.AppPlatform/Spring/myservice/configServers/default"),
+	// 	Properties: &armappplatform.ConfigServerProperties{
+	// 		ConfigServer: &armappplatform.ConfigServerSettings{
+	// 			GitProperty: &armappplatform.ConfigServerGitProperty{
+	// 				Label: to.Ptr("master"),
+	// 				SearchPaths: []*string{
+	// 					to.Ptr("/")},
+	// 					URI: to.Ptr("https://github.com/fake-user/fake-repository.git"),
+	// 				},
+	// 			},
+	// 			ProvisioningState: to.Ptr(armappplatform.ConfigServerStateSucceeded),
+	// 		},
+	// 	}
 }
 
-// Generated from example definition: https://github.com/Azure/azure-rest-api-specs/tree/main/specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2020-11-01-preview/examples/ConfigServers_UpdatePatch.json
+// Generated from example definition: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2020-11-01-preview/examples/ConfigServers_UpdatePatch.json
```

Comment:
I think last time we discussed about referencing the sample from a specific commit.
So here, why are we still using the one in main branch?

Reply:
Some of the tests are from the local swagger file that we could not find the commit. For those use public swagger repo, it will be referenced to a stable commit like this [test](https://github.com/Azure/autorest.go/blob/b585f2190764256c49c371c0ec90cc6ed2f796d0/packages/autorest.gotest/test/integrationtest/output/agrifood/armagrifood/extensions_client_example_test.go#L20).